pandas/core/nanops.py

```diff
@@ -1067,22 +1067,14 @@ def reduction(
         axis: AxisInt | None = None,
         skipna: bool = True,
         mask: npt.NDArray[np.bool_] | None = None,
-    ) -> Dtype:
-        dtype = values.dtype
+    ):
+        if values.size == 0:
+            return _na_for_min_count(values, axis)
```

Comment:
I assume we have tests that get here?

Reply:
The `values.size == 0` case is actually never reached in the current code:

You can see this function is wrapped in `bottleneck_switch` and if you go to `bottleneck_switch` you can see (line 125) it takes care of the `values.size == 0` case, also when `use_bottleneck = False` (`kwds.get("min_count") is None` is always true for `_nanminmax`).

I just think this is more readable than the old code.

Reply:
Gotcha. Thanks for the explanation